docs/demo/demo.md

```diff
@@ -262,6 +262,8 @@ in the trace changes.
 rm -rf $DEFAULT_VOCAB &&
   PYTHONPATH=$PYTHONPATH:. python3 \
     compiler_opt/tools/sparse_bucket_generator.py \
+    --gin_files=compiler_opt/rl/inlining/gin_configs/common.gin \
+    --gin_bindings=config_registry.get_configuration.implementation=@configs.InliningConfig \
```

Comment:
this is not needed cause it's in the first line of common.gin

Reply:
The `--gin_bindings` flag should also probably be removed on a couple other parts of the demo where it is just used to set the config. I can probably open up a PR to fix that, but it isn't a super critical issue.